tools/wptserve/wptserve/config.py

```diff
@@ -209,6 +212,12 @@ def __setattr__(self, key, value):
         else:
             self.__dict__[key] = value
 
+    def __getattr__(self, key):
+         if not key[0] == "_":
+             return self._data[key]
+         else:
+             return self.__dict__[key]
```

Comment:
What motivates this?

Reply:
The implementation is just based on the `__setattr__` definition. The reason for implementing the method at all is that we want to be able to override `config.logging` properties, and we can't do that without a getter (or reaching directly in to the inner data structures).

Reply:
When do we end up with anything on `__dict__` here anyway?

Reply:
Well the attributes set in `__init__` are all `_foo`, and so end up being instance properties.

Reply:
But don't instance properties bypass `__getattr__` anyway, because `__getattribute__` returns them?

Reply:
Ah yeah, good point. So it can just look in _data.